src/components/content-section/content-section.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
```

Comment:
@DanielStamper Missing this part:

```suggestion
import { ContentDiv } from './styles.js';
```